sources/engine/Xenko.Physics/HeightfieldTypes.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Xenko contributors (https://xenko.com)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Xenko.Physics
+{
+    public enum HeightfieldTypes
+    {
+        Short,
+        Byte,
```

Comment:
I think it might be better to have them sorted in size order. First `Byte` (8 bits), then `Short` (16 bits) and finally `float`(32 bits).

Reply:
That's right. Thanks you!

Reply:
Another idea I have: set their value to 1, 2 and 4 respectively.
That way we can get the size by just multiplying by 8. Might be useful some day.

Reply:
OK!